.github/workflows/pull_request_verify.yaml

```diff
@@ -0,0 +1,158 @@
+name: Pull Requests
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  CARGO_TARGET_DIR: ${{ github.workspace }}/../target
+  LOGBAK_USER: ${{secrets.LOGBAK_USER}}  #LOCBAK_* for upload logs to server when test failed, for windows
+  LOGBAK_PASSWORD: ${{secrets.LOGBAK_PASSWORD}}
+  LOGBAK_SERVER: ${{secrets.LOGBAK_SERVER}}
+jobs:
+  UnitTest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [Linux-runner, mac-runner,win-runner]
+        include:
+          - os: Linux-runner
+            script_run: make test
+          - os:  mac-runner
+            script_run: make test
+          - os: win-runner
+            script_run: devtools/windows/make test
+            CI: true
+            ImageOS: 'windows-2019'
+            BUILD_BUILDID: ${{ github.sha }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: UnitTest
+      run: ${{ matrix.script_run }}
+    env:
+      CI: ${{ matrix.CI }}
+      ImageOS:  ${{ matrix.ImageOS }}
+
+  Integration_Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [Linux-runner, mac-runner,win-runner]
+        include:
+          - os: Linux-runner
+            script_run: make integration
+          - os:  mac-runner
+            script_run: make integration
+          - os: win-runner
+            script_run: devtools/windows/make CKB_TEST_SEC_COEFFICIENT=5 CKB_TEST_ARGS="-c 4 --no-report" integration
+            CI: true
+            SENTRY_DSN: "https://15373165fbf2439b99ba46684dfbcb12@sentry.nervos.org/7"
+    needs: UnitTest
```

Comment:
These jobs can run in parallel. `Integration_Test` does not need to wait for `UnitTest`.

Reply:
Modified it, the jobs are queued for processing although these jobs can run in parallel.